src/modules/tests/eyesight-test-result/eyesight-test-result.selector.ts

```diff
@@ -2,3 +2,4 @@ import { EyesightTestResult } from '@dvsa/mes-test-schema/categories/B';
 
 export const isPassed = (result: EyesightTestResult) => result === 'P';
 export const isFailed = (result: EyesightTestResult) => result === 'F';
+export const getEyesightTestState = (result: EyesightTestResult) => result;
```

Comment:
This is just an identity function, why is it needed?

Reply:
Have modified as discuessed